tex/paper.tex

```diff
@@ -36,17 +36,36 @@
 
 \section{Introduction}\label{sec:introduction}
 
+Several iterated prisoner's dilemma tournaments have generated much interest,
+including Axelrod's original tournaments, two 2004 anniversary tournaments, and
+the S&P 2012 tournament following the discovery of zero-determinant strategies.
+Subsequent research has spawned an enormous number of papers, but rarely are the
+results reproducible. Among well-known tournaments, in only one case is the full
+original source code available (Axelrod's second tournament, in FORTRAN); in no
+cases is the available code well-documented, easily modifiable, or released
+with significant test suites.
+
+To make matters more complicated, often a newly-created strategy is studied in
+isolation by opponents chosen by the strategy's creators, and often such
+strategies are not sufficiently described to enable reliable recreation
+(in the absense of source code), with \cite{slany2007some} being a notable
+counter-example. In some cases strategies are revised without updates to their
+names or published implementations \cite{li2007design} \cite{li2011engineering}.
```

Comment:
\cite{li2007design, li2011engineering}

(Not necessarily telling anyone to do this, just making a note).